sensing/src/main/java/com/google/android/sensing/db/impl/entities/ResourceInfoEntity.kt

```diff
@@ -49,20 +50,26 @@ internal data class ResourceInfoEntity(
   val captureId: String,
 
   /** Participant for which the capture created this record. */
-  val participantId: String,
+  val externalIdentifier: String,
 
-  /** Title of the capture that creates this record. */
-  val captureTitle: String,
+  /** Android location of the captured data from a sensor. */
+  val localLocation: String,
 
-  /** Resource extension */
-  val fileType: String,
+  /** Absolute and immutable remote location, eg, url. */
+  val remoteLocation: String,
 
-  /** Relative android folder location of the captured data from a sensor. */
-  val resourceFolderRelativePath: String,
+  /** Label to display in place of the data. */
+  val resourceTitle: String,
 
-  /** Absolute and immutable upload url. */
-  val uploadURL: String,
+  /** Mime type of the content, with charset etc. */
+  val contentType: String,
+
+  /** Date this resource was first created. */
+  val creation: Instant,
 
   /** Upload status. */
   val status: RequestStatus,
+
+  /** The last time the [status] was updated. */
+  val lastUpdateTime: Instant
```

Comment:
On update of the record, this value should also be updated to current Instant value..

Reply:
Done.